css/custom.scss

```
@@ -330,4 +330,37 @@ $link-active: $link-hover;
 
 .peoples {
   margin-top: 1em;
-}
\ No newline at end of file
+}
+
+.button {
```

```suggestion
.button {
 a {color:#fff}
```

Hopefully better contrast for the text